src/components/TempTodo.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  tempTodo: Todo,
+};
+
+export const TempTodo: React.FC<Props> = ({
```

Comment:
You have a `TodoItem` component with the same design, use it.
As in your case if you have any design changes you have to keep in mind and change in two components.